client/my-sites/plans-features-main/index.jsx

```diff
@@ -209,7 +228,14 @@ class PlansFeaturesMain extends Component {
 	}
 
 	getFAQ() {
-		const { site, translate } = this.props;
+		const { isChatAvailable, site, translate } = this.props;
+
+		const helpLink =
```

Comment:
This `helpLink` and the one in `getJetpackFAQ()` are almost the same with the exception of the link href being different, so how do you feel about adding a method for this one?

Reply:
Thanks for the suggestion. I did consider this and implemented it but the result was actually more LOC between the unique feature check and href. I'm going to leave it.